src/workspace-data/data-table/wds/attribute-utils.js

```diff
@@ -0,0 +1,30 @@
+import * as Utils from 'src/libs/utils';
+
+export const getAttributeType = (attributeName, entityTypeAttributes, attributeValue, dataProvider) => {
+  // type that comes from the database schema in wds
+  let getTypeAttribute = entityTypeAttributes.find((attribute) => attribute.name === attributeName).datatype;
+
+  let isList = false;
+  if (getTypeAttribute.includes('ARRAY')) {
+    isList = true;
+    getTypeAttribute = getTypeAttribute.replace('ARRAY_OF_', '');
+  }
+
+  // type that the Attribute Value is holding
+  const type = Utils.cond(
+    // explicit double-equal to check for null and undefined, since entity attribute lists can contain nulls
+    // eslint-disable-next-line eqeqeq
+    [(isList ? attributeValue.items[0] : attributeValue) == undefined, () => 'string'],
+    [isList, () => typeof attributeValue.items[0]],
+    [typeof attributeValue === 'object', () => 'json'],
+    () => typeof attributeValue
+  );
+
+  // if the type matches, editing is allowed
+  if (type === getTypeAttribute.toLowerCase() && dataProvider.features.supportEntityUpdatingTypes.includes(getTypeAttribute.toLowerCase())) {
+    return { type: getTypeAttribute.toLowerCase(), isList };
+  }
+
+  // there is likely a better way to expose this
+  return { error: 'Editing the current type is not supported.' };
```

Comment:
I could not figure out an easy way to do this in the data table view (i.e. making the edit button not clickable and show a message with "type not supported") open to suggestions on how to do this better. This is the last field I try to edit in my video, it open the modal but it tells you the value cant be edited

Reply:
Can you remove the edit button completely from that datatype?

Reply:
@bethsheets  figured out how to do it, would you rather not have it all or keep it disabled with a message?
![image](https://github.com/DataBiosphere/terra-ui/assets/9418602/dc6d325b-bd8e-4861-9e08-94bb32474b76)

Reply:
I like this! I might update the language to "Editing this data type is not currently supported."